runtime/src/bank.rs

```diff
@@ -8010,6 +8010,17 @@ impl Bank {
                 loaded_programs_cache.program_runtime_environment_v1 =
                     Arc::new(program_runtime_environment_v1);
             }
+            let program_runtime_environment_v2 =
+                solana_loader_v4_program::create_program_runtime_v2_environment(
+                    &self.runtime_config.compute_budget.unwrap_or_default(),
+                    false, /* deployment */
```

Comment:
This is the parameter `debugging_features` not `deployment`, right?

Reply:
Yes, sorry, copypasta error on the comment.